packages/pieces/community/pdf-to-text/project.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "pieces-pdf-to-text",
```

Comment:
Can you regenerate the piece and call it `PDF` we can extend it to many other actions.

Reply:
Deleted current piece regenerated a new one